src/main/daml/Daml/Finance/Instrument/StructuredProduct/Util.daml

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Instrument.StructuredProduct.Util where
+
+import Daml.Finance.Data.Time.DateClock.Types (Unit(..))
+import Daml.Finance.Interface.Types.Date.Classes (toUTCTime)
+
+-- | Maps a `Date` to `Time` using the rule in the `DateClock`.
+dateToDateClockTime : Date -> Time
+dateToDateClockTime = toUTCTime . Unit
```

Comment:
Can you use the util defined in `D.F.Data.Time.DateClock`?

Reply:
Also, I believe this is duplicated in the `Option` package as well

Reply:
I found a few more duplicates as well. Replaced them all now.